go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -112,7 +97,7 @@ func (api *EthereumAPI) GetTransactionReceipt(_ context.Context, encryptedParams
 // EstimateGas is a placeholder for an RPC method required by MetaMask/Remix.
 func (api *EthereumAPI) EstimateGas(_ context.Context, _ interface{}, _ *rpc.BlockNumberOrHash) (hexutil.Uint64, error) {
 	// TODO - Return a non-dummy gas estimate.
-	return 0, nil
+	return 5_000_000_000, nil
```

Comment:
@otherview Can we revert this one for now?

Reply:
This is actually a pain point.

Having 0 means I need to find all Tx calls when using hardhat and manually hardcode them to some gas.
At the same time I must do the same when doing transactions with MetaMask.

I believe it's better to have it set at a high number rather than 0.
At that's the experience I'm having with HH and MM. WDYT ?